Newsletter-215: Translate to Hindi

shaavan opened the pull request.

bitschmidty commented on _posts/hi/newsletters/2022-08-31-newsletter.md:
```diff
@@ -0,0 +1,140 @@
+
+- [Eclair #2387][] [signet][topic signet] के लिए समर्थन जोड़ता है।
+
+- [LDK #1652][][onion message] के लिए अद्यतन समर्थन [topic onion messsage] * उत्तर पथ *
```
@shaavan the [onion message] ...... [topic onion messsage] needs to be corrected so it is clear what the link is and what is the link text. Im unsure of the language so need your help with that.

Also note "onion message" vs "onion mess**s**age"

shaavan replied:
Hi @bitschmidty, I was on a mini break, so I got your message today. Apologies for the late reply.

I figured the link tag was not correctly placed; hence the problem was not detected while running the `make production -j8`. There was also a similar mistake following it. So I have corrected both of them.
Thanks, @bitschmidty, for catching them!

shaavan commented:
Updated from 8b0a023ef9a4ed34f207ba98f44ff523e19ca4d3 to 7af392b4d5b17e634725b2d0402144040674e4ad ([pr850.01][https://github.com/shaavan/bitcoinops.github.io/commits/pr850.01] -> [pr850.02][https://github.com/shaavan/bitcoinops.github.io/commits/pr850.02], [diff][https://github.com/shaavan/bitcoinops.github.io/compare/pr850.01..pr850.02])

Changes:
- Fixed two mistakes with `[link tag][link]` not correctly written. Thank you @bitschmidty for catching it!

bitschmidty reviewed (approved):
Looks good, thanks @shaavan !
